### PR TITLE
[MODORG-56] - Update RMB and vertx to the latest version

### PR DESCRIPTION
**Pull request by Abdulkhakimov**
## Learning
[MODORG-56](https://issues.folio.org/browse/MODORG-56)

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate action.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code are 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added, or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all the appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally, all the PRs involved in breaking changes would be merged on the same day 
to avoid breaking the folio-testing environment.
Communication is paramount if that is to be achieved,
especially as the number of inter-module and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, 
ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by imerabishvili — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -14,20 +14,39 @@
...
     <jsonschema2pojo_output_dir>${generated_sources_dir}/jsonschema2pojo</jsonschema2pojo_output_dir>
-    <log4j.version>2.17.2</log4j.version>
+    <jsonschema_paths>acq-models/mod-orgs/schemas,raml-util/schemas,acq-models/common/schemas,acq-models/acquisitions-unit/schemas</jsonschema_paths>
+
+    <!--Dependency Management Properties-->
+    <log4j.version>2.20.0</log4j.version>
+
+    <!--Dependency properties-->
+    <aspectj.version>1.9.20.1</aspectj.version>
+    <raml-module-builder.version>35.1.0</raml-module-builder.version>
```
Comment (SerhiiNosko) [inline pom.xml:30]: seems we could not use 35.1.0 based on statement in this ticket https://issues.folio.org/browse/RMB-983: "RMB versions before 35.1.0 are not patched" and so we did not make any pathces in Poppy.
Now we are going to increase and 35.1.0 that contains some vulnerability, so we need to increase to 35.1.1 and Vertx 4.4.6

File: pom.xml [hunk 2]
```diff
@@ -14,20 +14,39 @@
...
+    <wiremock.version>2.26.3</wiremock.version>
+
+    <!--Maven plugin properties-->
+    <aspectj-maven-plugin.version>1.13.1</aspectj-maven-plugin.version>
+    <exec-maven-plugin.version>3.1.0</exec-maven-plugin.version>
+    <maven-resources-plugin.version>3.3.1</maven-resources-plugin.version>
+    <maven-compiler-plugin.version>3.11.0</maven-compiler-plugin.version>
+    <maven-antrun-plugin.version>3.1.0</maven-antrun-plugin.version>
+    <maven-shade-plugin.version>3.5.1</maven-shade-plugin.version>
+    <properties-maven-plugin.version>1.0.0</properties-maven-plugin.version>
```
Comment (SerhiiNosko) [inline pom.xml:44]: In the latest tag of RMB version of properties-maven-plugin is 1.2.0, so lets use the same
https://github.com/folio-org/raml-module-builder/blob/v35.1.1/pom.xml#L342

File: pom.xml (hunk 1 above)
Comment (Abdulkhakimov) [reply pom.xml]: Thanks for review. Done

File: pom.xml (hunk 2 above)
Comment (Abdulkhakimov) [reply pom.xml]: Done

**Review by SerhiiNosko — APPROVED** [review]

**Pull request merged**